src/Conversion/ONNXToTOSA/Math/Conv2D.cpp

```diff
@@ -91,40 +94,34 @@ class ONNXConvOpLoweringToTOSA : public OpConversionPattern<ONNXConvOp> {
     } else {
       // Set up constants outside of loop
```

Comment:
This is quite a hefty `else` clause and it feels like it deserves its own function with enough room to comment what it aims to produce.